lib/Connector.js

```diff
@@ -133,15 +130,15 @@ var Connector = {
 				body = body.substring(0, body.length - 1);
 			}
 		}
-		
+
 		var request_options = {
 			method: this.request_method,
 			url: url,
-			body: body
+			data: body

```

Comment:
If `body` includes `dataType: "jsonp"`, that might resolve the CORS issue.

Reply:
@mthommes I had to replace `request` with `axios` as `request` had some issues with being bundled, and `axios` does not support jsonp.